packages/react-server-dom-webpack/src/__tests__/ReactFlightDOM-test.js

```diff
@@ -841,6 +841,10 @@ describe('ReactFlightDOM', () => {
         expectedGamesValue,
     );
 
+    if (gate(flags => flags.enableOwnerStacks)) {
```

Comment:
I was expecting `enableServerComponentLogs` but the behavior is not actually behind this flag. Using `enableServerComponentLogs` would fail CI.